label_studio/tests/data_export.tavern.yml

```diff
@@ -104,6 +104,14 @@ stages:
       method: GET
     response:
       status_code: 200
+  - name: make conversion
```

Comment:
let's add download export CSV without convert step, it should be processed correctly with validation error and error code like 400/404.

Reply:
Added